package.json

```diff
@@ -21,45 +21,45 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.901.4",
-    "@angular/animations": "~9.1.4",
-    "@angular/cli": "~9.1.4",
-    "@angular/common": "~9.1.4",
-    "@angular/compiler": "~9.1.4",
-    "@angular/compiler-cli": "~9.1.4",
-    "@angular/core": "~9.1.4",
-    "@angular/forms": "~9.1.4",
-    "@angular/language-service": "~9.1.4",
-    "@angular/localize": "^9.0.7",
-    "@angular/platform-browser": "~9.1.4",
-    "@angular/platform-browser-dynamic": "~9.1.4",
-    "@angular/router": "~9.1.4",
+    "@angular-devkit/build-angular": "^0.901.12",
```

Comment:
(minor) need to be aware this has a potential semver chasnge - `~` to `^`. Does not affect build currently.